src/main/webapp/dev/js/common/assert.es6

```diff
@@ -0,0 +1,19 @@
+class AssertionFailedError extends Error {
+}
+
+function assert(condition, message) {
+    const msg = message === undefined || message === null ? 'Assertion Failed' : message;
+    if (!condition) {
+        throw new AssertionFailedError(msg);
```

Comment:
For some reason I had a natural aversion to doing that at first... But it really does look cleaner. :laughing: